stable/admin/templates/_helpers.tpl

```diff
@@ -376,4 +377,50 @@ release: {{ .Release.Name | quote }}
 {{- range $k, $v := .Values.admin.resourceLabels }}
 "{{ $k }}": "{{ $v }}"
 {{- end }}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+{{/*
+Renders an ephemeral volume for an AP.
+*/}}
+{{- define "admin.ephemeralVolume" -}}
+{{- if eq (include "defaultfalse" .Values.admin.ephemeralVolume.enabled) "true" }}
+ephemeral:
+  volumeClaimTemplate:
+    metadata:
+      labels:
+        {{- include "admin.resourceLabels" . | nindent 8 }}
+    spec:
+      accessModes:
+      - ReadWriteOnce
+      {{- if .Values.admin.ephemeralVolume.storageClass }}
+      {{- if (eq "-" .Values.admin.ephemeralVolume.storageClass) }}
```

Comment:
Should this now check for `nil`?

Reply:
I believe that's what `{{- if .Values.admin.ephemeralVolume.storageClass }}` does.